tests/proofs/locals-spec.k

```diff
@@ -3,7 +3,7 @@ requires "kwasm-lemmas.md"
 module LOCALS-SPEC
     imports KWASM-LEMMAS
 
-    rule <k> (local.get X:Int) (local.set X:Int) => . ... </k>
+    rule <k> sequenceInstrs((local.get X:Int) (local.set X:Int)) => . ... </k>
```

Comment:
I'd prefer to just sequence them directly with `~>` here, but up to you.

Reply:
Done